Update assignedaccess-csp.md (#3380)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/assignedaccess-csp.md

> [!Warning]
> You can only assign one single app kiosk profile to an individual user account on a device. The single app profile does not support domain groups.

> [!Note]

> > If the application runs in assigned access mode, when the app calls KeyCredentialManager.IsSupportedAsync and it returns false on the first run, try invoking the settings screen to have the user select a convenience PIN to use with Windows Hello. This is the settings screen that is hidden by the application running in assigned access mode. This means you can only use Windows Hello if you first leave Assigned Access. The user must then select his/her convenience pin and then go into Assigned Access again. 

Reviewer: ```suggestion
> If the application calls KeyCredentialManager.IsSupportedAsync when it is running in assigned access mode and it returns false on the first run, invoke the settings screen and select a convenience PIN to use with Windows Hello. This is the settings screen that is hidden by the application running in assigned access mode. You can only use Windows Hello if you first leave assigned access mode, select your convenience pin, and then go back into assigned access mode again. 
```
